mne/io/brainvision/brainvision.py

```diff
@@ -296,9 +298,18 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale, montage):
                       "not the '%s' file." % ext)
     with open(vhdr_fname, 'rb') as f:
         # extract the first section to resemble a cfg
-        header = f.readline().decode('utf-8').strip()
+        header = f.readline()
+        try:
+            header = header.decode('utf-8').strip()
+        except UnicodeDecodeError:
+            header = header.decode('ISO-8859-1')
```

Comment:
Also, I prefer ISO standards, but I guess not everybody knows them, so it's now Latin-1 ...